modules/core/src/main/scala/org/tessellation/infrastructure/snapshot/GlobalSnapshotConsensusFunctions.scala

```diff
@@ -117,15 +117,10 @@ object GlobalSnapshotConsensusFunctions {
         lastActiveTips <- lastArtifact.activeTips
         lastDeprecatedTips = lastArtifact.tips.deprecated
 
-        facilitators = lastArtifact.proofs.map(_.id)
-        transactions = lastArtifact.value.blocks.flatMap(_.block.transactions.toSortedSet).map(_.value)
+        transactionsForAcceptance = blocksForAcceptance.flatMap(_.value.transactions.toSortedSet).toSortedSet
+
+        rewardTxsForAcceptance <- rewards.distribute(lastArtifact, transactionsForAcceptance, trigger)
```

Comment:
Fee should be dristributed among facilitators from accepted transactions, not from transactions that maybe are going to be accepted or maybe not.

Reply:
fixed